### ISSUE TITLE
Restoring app via Helium stops notifications from occurring

### ISSUE THREAD

**Issue by neomodern** [vector-im/riot-android#887]
I use Helium to backup & restore apps and it was able to successfully restore 80+ apps after I migrated phones. The only problem was Riot - it recovered my app data just fine but I never seemed to get any notifications from it anymore once the app was closed.

Checked the notifications settings (on Android + rooms) which was fine, then discovered it always gave me new notifications only once I actually opened the app and kept it open.

Clearing the cache didn't work, signing out and logging back in didn't work, it took a full reinstall to get the push notifications back once the app was left closed. 

Obviously an edge case but it may highlight deeper structural inefficiencies within the app.

**Comment by ylecollen** [comment]
duplicate of https://github.com/vector-im/riot-android/issues/725
